src/globals.cpp

```diff
@@ -30,8 +30,9 @@ namespace Globals {
 int nghost;
 
 // all of these global variables are set at the start of main():
-int my_rank; // MPI rank of this process
-int nranks;  // total number of MPI ranks
+int my_rank;     // MPI rank of this process
+int nranks;      // total number of MPI ranks
+bool is_restart; // Whether this simulation is restarted from a checkpoint file
```

Comment:
Slight preference for `Globals::restart` over `Globals::is_restart`

Reply:
`bool is_*` is a fairly common pattern in parthenon, and since this variable won't be appearing regularly in complex expressions I'm inclined to leave as is unless a majority pushes back on this

Reply:
I don't feel strongly. I think I have a slight preference for `bool is_*`.